schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -318,9 +318,14 @@
                     "description": "the date/time this went public if known"
                 },
                 "replacedBy": {
-                    "type": "string",
-                    "description": "a single CVE ID or list of CVE IDs (comma separated)",
-                    "pattern": "^(CVE-[0-9]{4}-[0-9]{4,})\\s*(,\\s*CVE-[0-9]{4}-[0-9]{4,})*$"
+                    "type": "array",
+                    "$comment": "TODO: improve this definition",
+                    "description": "an array of CVE IDs",
```

Comment:
The old description didn't say anything about what this field means semantically. I'm not completely sure either. Can someone provide a definition?